core/src/test/java/com/sequenceiq/cloudbreak/converter/v2/StackToTemplatePreparationObjectConverterTest.java

```diff
@@ -189,6 +190,9 @@
     @Mock
     private KerberosConfigService kerberosConfigService;
 
+    @Mock
+    private CustomConfigsService customConfigsService;
```

Comment:
you are mocking this class but no verification at all in the class. the invokation should be tested